Add script to fit NET vs. PWV

PaulKGrimes opened the pull request.

keskitalo committed:
Add script to fit NET vs. PWV

eacharles commented:
Hi Paul, just wanna make sure you have all the keys you need for this repo.

PaulKGrimes commented:
Hi, yes, I've been able to commit branches, etc. to the repo.

I'm working some substantial updates to the SAT models, bringing the PBDR
runs for SATs up to v4 (HDPE optics at HF), and then on AoA for the SATs
team, so I'm working in branches for now.  I have made some improvements to
the various Jupyter notebooks, particularly the Comparisons one - which can
now compare multiple models.

In parallel, I've had to make some fixes to bolo-calc, to get it to run
correctly on my Windows Anaconda set ups, and to distribute the PKL
aperture files with the package.  Those are all tested on both linux and
windows, and I have a pull request open on that repo.

Thanks,

Paul

On Thu, May 26, 2022 at 4:49 PM Eric Charles ***@***.***>
wrote:

> Hi Paul, just wanna make sure you have all the keys you need for this repo.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CMB-S4/bolo_calc_runs/pull/1#issuecomment-1139028285>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGT6ZSLE6D44CITRRLPOVGLVL7PT7ANCNFSM5W26WWMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Receiver Laboratory
Smithsonian Astrophysical Observatory
60 Garden St, Cambridge, MA 02138
+1 (617) 496-7640

eacharles commented:
Do you need the keys for bolo-calc?  I think it is in KIPAC space for historical reasons.

-e

> On May 26, 2022, at 1:57 PM, Paul Grimes ***@***.***> wrote:
> 
> 
> Hi, yes, I've been able to commit branches, etc. to the repo.
> 
> I'm working some substantial updates to the SAT models, bringing the PBDR
> runs for SATs up to v4 (HDPE optics at HF), and then on AoA for the SATs
> team, so I'm working in branches for now. I have made some improvements to
> the various Jupyter notebooks, particularly the Comparisons one - which can
> now compare multiple models.
> 
> In parallel, I've had to make some fixes to bolo-calc, to get it to run
> correctly on my Windows Anaconda set ups, and to distribute the PKL
> aperture files with the package. Those are all tested on both linux and
> windows, and I have a pull request open on that repo.
> 
> Thanks,
> 
> Paul
> 
> On Thu, May 26, 2022 at 4:49 PM Eric Charles ***@***.***>
> wrote:
> 
> > Hi Paul, just wanna make sure you have all the keys you need for this repo.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/CMB-S4/bolo_calc_runs/pull/1#issuecomment-1139028285>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AGT6ZSLE6D44CITRRLPOVGLVL7PT7ANCNFSM5W26WWMQ>
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
> 
> 
> -- 
> Receiver Laboratory
> Smithsonian Astrophysical Observatory
> 60 Garden St, Cambridge, MA 02138
> +1 (617) 496-7640
> —
> Reply to this email directly, view it on GitHub <https://github.com/CMB-S4/bolo_calc_runs/pull/1#issuecomment-1139041184>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ADRIGIU7TZASKO2T5GVCHQLVL7QR3ANCNFSM5W26WWMQ>.
> You are receiving this because you commented.
>

eacharles commented:
Ok, I invited you to the bolo-calc repo.  Feel free to do what ever you need.

The github actions might be useful, e.g., for automated linting and it at least runs a couple of simple tests.

-e

> On May 26, 2022, at 2:01 PM, Eric Charles ***@***.***> wrote:
> 
> Do you need the keys for bolo-calc?  I think it is in KIPAC space for historical reasons.
> 
> -e
> 
>> On May 26, 2022, at 1:57 PM, Paul Grimes ***@***.*** ***@***.***>> wrote:
>> 
>> 
>> Hi, yes, I've been able to commit branches, etc. to the repo.
>> 
>> I'm working some substantial updates to the SAT models, bringing the PBDR
>> runs for SATs up to v4 (HDPE optics at HF), and then on AoA for the SATs
>> team, so I'm working in branches for now. I have made some improvements to
>> the various Jupyter notebooks, particularly the Comparisons one - which can
>> now compare multiple models.
>> 
>> In parallel, I've had to make some fixes to bolo-calc, to get it to run
>> correctly on my Windows Anaconda set ups, and to distribute the PKL
>> aperture files with the package. Those are all tested on both linux and
>> windows, and I have a pull request open on that repo.
>> 
>> Thanks,
>> 
>> Paul
>> 
>> On Thu, May 26, 2022 at 4:49 PM Eric Charles ***@***.***>
>> wrote:
>> 
>> > Hi Paul, just wanna make sure you have all the keys you need for this repo.
>> >
>> > —
>> > Reply to this email directly, view it on GitHub
>> > <https://github.com/CMB-S4/bolo_calc_runs/pull/1#issuecomment-1139028285 <https://github.com/CMB-S4/bolo_calc_runs/pull/1#issuecomment-1139028285>>,
>> > or unsubscribe
>> > <https://github.com/notifications/unsubscribe-auth/AGT6ZSLE6D44CITRRLPOVGLVL7PT7ANCNFSM5W26WWMQ <https://github.com/notifications/unsubscribe-auth/AGT6ZSLE6D44CITRRLPOVGLVL7PT7ANCNFSM5W26WWMQ>>
>> > .
>> > You are receiving this because you authored the thread.Message ID:
>> > ***@***.***>
>> >
>> 
>> 
>> -- 
>> Receiver Laboratory
>> Smithsonian Astrophysical Observatory
>> 60 Garden St, Cambridge, MA 02138
>> +1 (617) 496-7640
>> —
>> Reply to this email directly, view it on GitHub <https://github.com/CMB-S4/bolo_calc_runs/pull/1#issuecomment-1139041184>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ADRIGIU7TZASKO2T5GVCHQLVL7QR3ANCNFSM5W26WWMQ>.
>> You are receiving this because you commented.
>> 
>

JohnRuhl committed:
adding AoA models

JohnRuhl committed:
moved recent results to new directory

JohnRuhl committed:
updated SAT yamls to reflect correct layout, horn diameters, in MFs

JohnRuhl committed:
adding some notebooks with band and atmos investigations in jr_runs directory

JohnRuhl committed:
fixing jr_runs by adding symlinks

PaulKGrimes commented:
Thanks!

I'll try not to break anything :)

Paul

On Thu, May 26, 2022 at 5:14 PM Eric Charles ***@***.***>
wrote:

> Ok, I invited you to the bolo-calc repo. Feel free to do what ever you
> need.
>
> The github actions might be useful, e.g., for automated linting and it at
> least runs a couple of simple tests.
>
> -e
>
> > On May 26, 2022, at 2:01 PM, Eric Charles ***@***.***> wrote:
> >
> > Do you need the keys for bolo-calc? I think it is in KIPAC space for
> historical reasons.
> >
> > -e
> >
> >> On May 26, 2022, at 1:57 PM, Paul Grimes ***@***.*** ***@***.***>>
> wrote:
> >>
> >>
> >> Hi, yes, I've been able to commit branches, etc. to the repo.
> >>
> >> I'm working some substantial updates to the SAT models, bringing the
> PBDR
> >> runs for SATs up to v4 (HDPE optics at HF), and then on AoA for the SATs
> >> team, so I'm working in branches for now. I have made some improvements
> to
> >> the various Jupyter notebooks, particularly the Comparisons one - which
> can
> >> now compare multiple models.
> >>
> >> In parallel, I've had to make some fixes to bolo-calc, to get it to run
> >> correctly on my Windows Anaconda set ups, and to distribute the PKL
> >> aperture files with the package. Those are all tested on both linux and
> >> windows, and I have a pull request open on that repo.
> >>
> >> Thanks,
> >>
> >> Paul
> >>
> >> On Thu, May 26, 2022 at 4:49 PM Eric Charles ***@***.***>
> >> wrote:
> >>
> >> > Hi Paul, just wanna make sure you have all the keys you need for this
> repo.
> >> >
> >> > —
> >> > Reply to this email directly, view it on GitHub
> >> > <
> https://github.com/CMB-S4/bolo_calc_runs/pull/1#issuecomment-1139028285 <
> https://github.com/CMB-S4/bolo_calc_runs/pull/1#issuecomment-1139028285>>,
> >> > or unsubscribe
> >> > <
> https://github.com/notifications/unsubscribe-auth/AGT6ZSLE6D44CITRRLPOVGLVL7PT7ANCNFSM5W26WWMQ
> <
> https://github.com/notifications/unsubscribe-auth/AGT6ZSLE6D44CITRRLPOVGLVL7PT7ANCNFSM5W26WWMQ
> >>
> >> > .
> >> > You are receiving this because you authored the thread.Message ID:
> >> > ***@***.***>
> >> >
> >>
> >>
> >> --
> >> Receiver Laboratory
> >> Smithsonian Astrophysical Observatory
> >> 60 Garden St, Cambridge, MA 02138
> >> +1 (617) 496-7640
> >> —
> >> Reply to this email directly, view it on GitHub <
> https://github.com/CMB-S4/bolo_calc_runs/pull/1#issuecomment-1139041184>,
> or unsubscribe <
> https://github.com/notifications/unsubscribe-auth/ADRIGIU7TZASKO2T5GVCHQLVL7QR3ANCNFSM5W26WWMQ
> >.
> >> You are receiving this because you commented.
> >>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CMB-S4/bolo_calc_runs/pull/1#issuecomment-1139064202>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGT6ZSK4E3MZJ7W37TYE7LLVL7SVDANCNFSM5W26WWMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Receiver Laboratory
Smithsonian Astrophysical Observatory
60 Garden St, Cambridge, MA 02138
+1 (617) 496-7640

JohnRuhl committed:
PBDR_2023 dierectory added

JohnRuhl committed:
added BandOptimization notebook

JohnRuhl committed:
LF_band edges update, with synch sensitivity, and moving transition frequency between LF_1 and LF_2

JohnRuhl committed:
deleting LF_band_edges, b/c won't push

JohnRuhl committed:
readding LF_band_edges notebook

JohnRuhl committed:
test.txt

JohnRuhl committed:
remove test.txt

JohnRuhl committed:
updating pbdr calculations with current models

JohnRuhl committed:
LF band edges, fixed last plot

JohnRuhl committed:
getting PBDR straight, should be near final version

JohnRuhl committed:
adding toml outputs

JohnRuhl committed:
adding bk ratio stuff

JohnRuhl committed:
added psats toml, and notebook showing how nets change with varying module optical efficiency

JohnRuhl committed:
added 2D (elev,pwv) output toml file

JohnRuhl committed:
updated elev_pwv_2D file to include Pelec, which goes below zero when saturated

JohnRuhl committed:
Added 2D (elev,pwv) outputs

JohnRuhl committed:
updating AoA with previous work

JohnRuhl committed:
moved existing files to 2022 subdirectory

JohnRuhl committed:
aperture funcs moved, but symbolic link?

JohnRuhl committed:
adding AoA/2024

JohnRuhl committed:
writing mapping speed ratios

JohnRuhl committed:
new AoA 2024 SAT beam estimates

JohnRuhl committed:
Both LATs in Chile, notebook and yamls added

JohnRuhl committed:
updated AoA/2024 yamls and notebooks

JohnRuhl committed:
update to SAT NETs

JohnRuhl committed:
fixed bug found by Michel Piat, re mistake in keeping psats fixed.  Wrote out new tomls.

smsimon committed:
Add yaml file for Phase 2 case

SO-like design but with larger optics

smsimon committed:
Create test.txt

smsimon committed:
Noises run with Phase 2 configuration

Note that the Phase 2 configuration label in the toml file is "Chile_aggr"

smsimon committed:
Delete jbolo/AoA/2024/tomls_20241121/test.txt

smsimon committed:
Notebook used for Phase 2 Noise Calculations

smsimon committed:
Non-split band configuration (MF2 is the MF band)

Add non-split band configuration. MF2 field is the non-split MF band.

smsimon committed:
Non-split band configuration (MF2 is the MF band)

Add a phase 2 configuration with non-split bands (MF2 bands are the non-split MF2 bands)

JohnRuhl committed:
adding 2025 directory, with Chile-optimization, exploration, files

JohnRuhl committed:
SAT comparisons study

JohnRuhl committed:
fixing SAT_comparisions notebook

JohnRuhl committed:
SAT comparisons things

JohnRuhl committed:
cleanup/corrections to models

PaulKGrimes committed:
Compare thin window AoA alt 1 SATs at Pole using default 273K and BICEP3 estimated effective temperatures for forebaffle and window

PaulKGrimes committed:
Update to compare thin windows at Pole to previous 20mm HMPE window

PaulKGrimes committed:
Compare SAT design changes one-by-one

Carry out a comparison of the SAT design changes introduced in the Chile Optimization Report, introducing each change in the numerical order shown in the report
Repeat with thin windows.
(Uses BICEP/Keck measured forebaflle and window temperatures for Pole)
